server/middleware/populateCurrentUser.ts

```diff
@@ -25,20 +25,34 @@ export default function populateCurrentUser(userService: UserService): RequestHa
         userRoles: roles.map(role => role.substring(role.indexOf('_') + 1)),
       }
 
+      res.locals.agent = {
+        username: res.locals.user.username,
+        fullName: convertToTitleCase(name),
+        role: 'SUPPORT',
```

Comment:
better to not assume that role is support unless they're accessing our support pages